test/integration/scheduler/queue_test.go

```diff
@@ -75,10 +75,10 @@ func TestCoreResourceEnqueue(t *testing.T) {
 	// - Pod1 is a best-effort Pod, but doesn't have the required toleration.
 	// - Pod2 requests a large amount of CPU resource that the node cannot fit.
 	//   Note: Pod2 will fail the tainttoleration plugin b/c that's ordered prior to noderesources.
-	// - Pod3 has the required toleration, but requests a non-existing PVC.
```

Comment:
This test case was making Pod3 unschedulable with non-existing PVC. This Pod was rejected by volume binding plugin in PreFilter [here](https://github.com/kubernetes/kubernetes/blob/9bbc5b15e7dfb97bfa9a743262d5fc932d3606a6/pkg/scheduler/framework/plugins/volumebinding/volume_binding.go#L175) with `UnschedulableAndUnresolvable` status.

This PR enables non-csi-limit plugin in PreFilter and it is executed faster than volumebinding. And Pod3 is rejected by it in PreFilter with `Error` status. This change caused this test case to fail.

So, I changed this test case to use NodeAffinity.